src/BackupGenerator.php

```diff
@@ -0,0 +1,355 @@
+<?php
+
+namespace Blocktrail\SDK;
+
+use Blocktrail\SDK\Bitcoin\BIP32Key;
+use Endroid\QrCode\QrCode;
+
+/**
+ * Class BackupGenerator
+ */
+class BackupGenerator {
+
+    const QR_CODE_SIZE = 195;
+
+    /**
+     * path to fonts used for pdf generation
+     *
+     * @var string
+     */
+    protected $fontsPath;
+
+    /**
+     * BIP39 Mnemonic for the master primary private key
+     *
+     * @var string
+     */
+    protected $primaryMnemonic;
+
+    /**
+     * BIP39 Mnemonic for the backup private key
+
+     * @var string
+     */
+    protected $backupMnemonic;
+
+    /**
+     * map of blocktrail BIP32 public keys
+     *  keyed by key index
+     *  path should be `M / key_index'`
+     *
+     * @var BIP32Key[]
+     */
+    protected $blocktrailPublicKeys;
+
+    /**
+     * @param $primaryMnemonic
+     * @param $backupMnemonic
+     * @param $blocktrailPublicKeys
+     */
+    public function __construct($primaryMnemonic, $backupMnemonic, $blocktrailPublicKeys) {
+        if (!defined('DOMPDF_ENABLE_AUTOLOAD')) {
+            // disable DOMPDF's internal autoloader if you are using Composer
+            define('DOMPDF_ENABLE_AUTOLOAD', false);
+
+            // include DOMPDF's default configuration
+            require_once dirname(__FILE__) . '/../vendor/dompdf/dompdf/dompdf_config.inc.php';
+        }
+
+        //set the fonts path
+        $this->fontsPath = dirname(__FILE__) . '/../resources/fonts';
+
+        $this->primaryMnemonic = $primaryMnemonic;
+        $this->backupMnemonic = $backupMnemonic;
+
+        $blocktrailPublicKeys = array_map(function ($key) {
+            return BIP32Key::create($key); // M/9999' or M/0' or M/1'
+        }, $blocktrailPublicKeys);
+
+        //create QR codes for each blocktrail pub key
```

Comment:
don't 'do stuff' in a constructor, generally people will expect that a constructor just does the bare minimum to initialize and then call a method that 'does stuff'